### PR TITLE
Rspec Model

### PR DESCRIPTION
**Pull request by nampt-3121**
## Related Tickets
- [#51570](https://edu-redmine.sun-asterisk.vn/issues/51570)

## WHAT (optional)
- Rspec Model

## HOW
- Rspec for Comment Model 

## WHY (optional)
- Rspec Model

## Evidence (Screenshot or Video)
Screen Shot 2022-08-03 at 11 40 20
Screen Shot 2022-08-03 at 11 40 25

### CODE REVIEW

**Comment by nampt-3121** [comment]
ready

**Comment by nampt-3121** [comment]
ready

File: spec/models/report_spec.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  let!(:department) {FactoryBot.create :department, id: 1}
+  let!(:department_2) {FactoryBot.create :department, id: 2}
+  let!(:relationship_user) {FactoryBot.create :relationship, user_id: user.id, department_id: department.id, role_type: :employee}
+  let!(:relationship_manager) {FactoryBot.create :relationship, user_id: user_manager.id, department_id: department.id, role_type: :manager}
+  let!(:report_1) {FactoryBot.create :report, id: 1, from_user_id: user.id, to_user_id: user_manager.id, department_id: department.id}
+  let!(:report_2) {FactoryBot.create :report, id: 2, from_user_id: user.id, to_user_id: user_manager.id, department_id: department.id}
+  let!(:report_3) {FactoryBot.create :report, id: 3, from_user_id: user.id, to_user_id: user_manager.id, department_id: department.id, report_status: 1}
+  let!(:report_4) {FactoryBot.create :report, id: 4, from_user_id: user_singe.id, to_user_id: user_manager.id, department_id: department.id}
+  let!(:report_5) {FactoryBot.create :report, id: 5, from_user_id: user_singe.id, to_user_id: user_manager.id, department_id: department_2.id, report_date: "2022-02-02"}
```
Comment (ngocvt-0484) [inline spec/models/report_spec.rb]: dài quá e xuống dòng nhé

File: spec/models/report_spec.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
+require "rails_helper"
+
+RSpec.describe Report, type: :model do
+  let!(:user) {FactoryBot.create :user, id: 1}
+  let!(:user_manager) {FactoryBot.create :user, id: 2, role: :normal}
+  let!(:user_admin) {FactoryBot.create :user, id: 3, role: :admin}
+  let!(:user_singe) {FactoryBot.create :user, id: 4, role: :normal}
+  let!(:department) {FactoryBot.create :department, id: 1}
```
Comment (ngocvt-0484) [inline spec/models/report_spec.rb]: bỏ id đi

**Comment by nampt-3121** [comment]
ready

**Pull request merged**